samples/matter/template/boards/nrf54h20_suit.overlay

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Reorganize partitions for SUIT */
```

Comment:
Correct me, if I'm wrong, but do we really need an overlay? As far as I understand SUIT is required for the DFU purposes, so we don't want to support configuration without it. I think we should just add it to the common nrf54h20 board overlay.

Reply:
For now, we don't have implemented and tested DFU using external flash - it is planned in some weeks. Currently, we cannot perform dfu both for app + net core because we don't have enough space, so for development purposes, I would like to create an overlay that allows us to build the App with 98% occupancy but leave the original DTS for building the App with 68% occupancy. Once the external flash is supported it will be removed or changed.